app/models/ticket_comment.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class TicketComment < ApplicationRecord
+  belongs_to :ticket
+  belongs_to :acting_user, class_name: 'User', foreign_key: :acting_user_id
+
+  DEFAULT_SERIALIZE_OPTIONS = {
+    include: %w[acting_user],
+  }.freeze
+
+  def serializable_hash(options = nil)
+    json = super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
+    json[:class] = self.class.to_s.downcase
```

Comment:
If I'm not mistaken, this will take the class name `TicketComment`, and convert it to downcase like `ticketComment`. Why not `dasherize` or `underscore` or any other casing?

And on that note, why do you need to export the class name anyways?

Reply:
> I simply copied the thing from there without reading it completely.

😐 Copying from other places is totally fine, but only if you understand what you're copying!